### PR TITLE
updage connector options doc for table-api

### PR DESCRIPTION
**Pull request by hehuiyuan**
**Change log description**
Update doc for connector options.
Add scope option and scan.reader-group.name option

### CODE REVIEW

File: documentation/src/docs/table-api.md [hunk 1]
```diff
@@ -82,13 +82,14 @@ WITH (
...
 | security.validate-hostname                             | optional            | (none)        | Boolean      | If host name validation should be enabled when TLS is enabled                                                 |
 | security.trust-store                                   | optional            | (none)        | String       | Trust Store for Pravega client                                                                                |
 | scan.execution.type                                    | optional            | streaming     | String       | Execution type for scan source. Valid values are 'streaming', 'batch'.                                        |
-| scan.reader-group.name                                 | optional            | (none)        | String       | Pravega reader group name                                                                                     |
+| scan.reader-group.name                                 | required for source | (none)        | String       | Pravega reader group name                                                                                     |
```
Comment (crazyzhou) [inline documentation/src/docs/table-api.md:92]: This is changed to optional by #493 which is addressed in 0.9.1 release notes.

**Review by crazyzhou — CHANGES_REQUESTED** [review]
Thanks for your contribution, I have created issue #634 for this and please use this as the PR title and sign-off your commit.

**Comment by hehuiyuan** [comment]
亲，发的东西收到了！

**Comment by hehuiyuan** [comment]
亲，发的东西收到了！

**Comment by crazyzhou** [comment]
As there is no valid response for a long time, we will close this PR

**Comment by hehuiyuan** [comment]
亲，发的东西收到了！

**Comment by hehuiyuan** [comment]
> As there is no valid response for a long time, we will close this PR

Sorry for the delay

**Pull request closed without merging**